backend/src/api/controllers/result.ts

```diff
@@ -518,6 +520,23 @@ async function calculateXp(
     }
   }
 
+  console.log(funboxBonus);
+  console.log(funbox);
+  if (funboxBonus > 0) {
+    let funboxModifier = 0;
+    for (const fb of funbox.split("#")) {
+      const funbox = FunboxConstants.find((f) => f.name === fb);
+      const modifier = (funbox?.difficultyLevel ?? 0) * funboxBonus;
+      if (modifier > 0) {
+        funboxModifier += modifier;
+      }
+    }
```

Comment:
This whole function can be condensed to:

```suggestion
const funboxModifier = _.sumBy(funboxes.split("#"), (funboxName) => {
    const funbox = FunboxConstants[funboxName];
    
    return Math.max((funbox?.difficultyLevel ?? 0) * funboxBonus, 0);
});
```